tests/esp32/wait_for1.toit

```diff
@@ -26,8 +26,8 @@ ITERATIONS ::= 10_000
 main:
   pin_in := gpio.Pin PIN_IN --input --pull_down
   pin_out := gpio.Pin PIN_OUT --output
-  sleep --ms=100
 
+  pin_in.configure --input --pull_down
```

Comment:
This is due to another bug that I will (hopefully) fix next.

Reply:
Will you drop this again?

Reply:
done.